zhaquirks/lixee/zlinky.py

```diff
@@ -15,6 +16,19 @@
 )
 from zhaquirks.lixee import LIXEE
 
+ZLINKY_MANUFACTURER_CLUSTER_ID = 0xFF66
+
+
+class ZLinkyTICManufacturerCluster(CustomCluster):
+    """ZLinkyTICManufacturerCluster manufacturer cluster."""
+
+    cluster_id = ZLINKY_MANUFACTURER_CLUSTER_ID
+    name = "ZLinky_TIC Manufacturer specific"
+    ep_attribute = "zlinky_cluster"
+    manufacturer_attributes = {
+        0x0000: ("optarif_or_ngtf", t.LimitedCharString(16)),
```

Comment:
Should the attribute name be translated from French to English? Doing so would be harder to maintain.

Reply:
Hrm, @Adminiuga what do you think? My initial thought is yes and we could have a comment with the French translation on each line to help with the maintenance… the entire code base is in English so having just this in French would be a bit odd IMO

Reply:
Sounds good to me, I'll proceed with the translation.